etc/ci/check_intermittents.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Shebang should be `#!/usr/bin/env bash` per the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).